src/App.js

```diff
@@ -37,28 +37,34 @@ const NoLogin = () => {
  * Main App component. Wrap the main react-router components together with
  * the various dialogs and error messages that may be needed.
  */
-const App = ({ history, config, appReady }) => {
-  if (!config.get('loginToken')) { // login is missing
+
+class App extends React.Component {
+  render () {
+    const { history, config, appReady } = this.props
```

Comment:
No real `render()` change to `App` right?  Maybe just drop these changes since `App` doesn't need to be a React.Component class.

(I'd use `git gui` to ammend the last commit and unstage the change to the file)